test/contestant_test.rb

```diff
@@ -0,0 +1,65 @@
+require 'minitest/autorun'
+require './lib/contestant'
+require "./lib/game"
+require 'pry'
+
+
+class ContestantTest < Minitest::Test
+
+def test_instance_of_game
+  pick_4 = Game.new('Pick 4', 2)
+  assert_instance_of Game, pick_4
+end
```

Comment:
The ContestantTest should only have tests that revolve around the Contestant class - Game deserves its own test.